paasta_tools/cli/cmds/spark_run.py

```diff
@@ -56,6 +56,12 @@
 CLUSTER_MANAGER_MESOS = "mesos"
 CLUSTER_MANAGER_K8S = "kubernetes"
 CLUSTER_MANAGERS = {CLUSTER_MANAGER_MESOS, CLUSTER_MANAGER_K8S}
+# Reference: https://spark.apache.org/docs/latest/configuration.html#application-properties
+DEFAULT_DRIVER_CORES_BY_SPARK = 1
+DEFAULT_DRIVER_MEMORY_BY_SPARK = "1g"
```

Comment:
I'm actually worried in making this the default at the moment. There are a lot of batches that don't specify `spark.driver.cores` and/or `spark.driver.memory` that might fail if this change is pushed. What is the pushplan for this change?

Reply:
Do have the same concern as above

Reply:
These default values are the one provided by spark for the above configs. 
The behavior of the batch should be same irrespective of these configs not provided or provided with above values. 
Manual test results in the ticket.

tl;dr when batch doesn't provide these configs:
* Docker container memory limit as 2g
* Docker cpu limit same 1

My only concern is, if should have the core limit > the one set by spark driver.

Reply:
@gcollura , @chelspea let me know if I can resolve this comment?